config.properties.example

```diff
@@ -39,6 +39,17 @@ password=maxwell
 # name of the mysql database where maxwell keeps its own state
 #schema_database=maxwell
 
+# SSL/TLS options
+# To use any modes other than DISABLED, you must set the trust store:
+#   set JAVA_OPTS="-Djavax.net.ssl.trustStore=<truststore> -Djavax.net.ssl.trustStorePassword=<password>"
+# MODE must be one of DISABLED, PREFERRED, REQUIRED, VERIFY_CA, or VERIFY_IDENTITY
```

Comment:
so I am testing the code locally, it seems like it works fine with `--ssl=REQUIRED` without bothing to set any of the `trustStore` options -- maybe those are only required for `VERIFY_CA` and `VERIFY_IDENTITY`?

Reply:
I'll locally test as well - that makes sense, though. [here's](https://github.com/shyiko/mysql-binlog-connector-java/blob/master/src/main/java/com/github/shyiko/mysql/binlog/BinaryLogClient.java#L669) where the SSLMode is used in the shyiko binlog connector, and [here](https://github.com/shyiko/mysql-binlog-connector-java/blob/master/src/main/java/com/github/shyiko/mysql/binlog/BinaryLogClient.java#L90) are the certificate checking configs.

so for `REQUIRED` and `PREFERRED`, there aren't any checks that happen (those `TrustManager` methods are stubbed out), but for the higher ones, the default check is used.

Reply:
if that's the case, I can update the comments here to include that